src/components/BookingForm.vue

```diff
@@ -43,6 +43,7 @@
         </bad-contained-button>
       </v-col>
     </v-row>
+    <span>{{feedback}}</span>
```

Comment:
Maybe you could create a new component instead of the span? That way, we can change it later on.